src/framework/shortcuts/view/editmidimappingmodel.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "editmidimappingmodel.h"
+
+#include "log.h"
+#include "utils.h"
+#include "translation.h"
+
+using namespace mu::shortcuts;
+using namespace mu::midi;
+
+static const QString WAITING_STATE = mu::qtrc("shortcuts", "Waiting...");
```

Comment:
Same about the translations. And isn't it too much to use a named constant here? It's used only one time.